lib/ads/interstitial_ad_manager.js

```diff
@@ -502,21 +531,39 @@ shaka.ads.InterstitialAdManager = class {
       this.onEvent_(new shaka.util.FakeEvent(
           shaka.ads.Utils.AD_SKIP_STATE_CHANGED));
     }
+    const eventsSent = [];
```

Comment:
This should be a Set instead of an array.

Reply:
Done!